modulefiles/module_base.hera.lua

```diff
@@ -0,0 +1,35 @@
+help([[
+Load environment to run GFS on Hera
+]])
+
+prepend_path("MODULEPATH", "/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack")
+
+load(pathJoin("hpc", os.getenv("hpc_ver")))
+load(pathJoin("hpc-intel", os.getenv("hpc_intel_ver")))
+load(pathJoin("hpc-impi", os.getenv("hpc_impi_ver")))
+
+load(pathJoin("esmf", os.getenv("esmf_ver")))
+
+load(pathJoin("python", os.getenv("python_ver")))
+load(pathJoin("gempak", os.getenv("gempak_ver")))
+load(pathJoin("perl", os.getenv("perl_ver")))
```

Comment:
perl, libjpeg, udunits, bufr_dump, and util_shared weren't loaded in TCL version

Reply:
The `perl` module is for the GSI monitoring package.

The `libjpeg` module is a prereq for the `grib_util` module in the WCOSS2 stack. It's not listed as a prereq for the same `grib_util` module on either Hera or Orion. Asked Kyle to install `libjpeg` in the [same issue comment](https://github.com/NOAA-EMC/hpc-stack/issues/379#issuecomment-1033829965) as `udunits`.

The `udunits` is needed for something in the system on WCOSS2, although I'm having trouble finding the reason in my email. Hera has the same version as WCOSS2 and [I've already asked Kyle to add the same version to the stack on Orion](https://github.com/NOAA-EMC/hpc-stack/issues/379#issuecomment-1033829965).

The `bufr_dump` module is needed by the WAFS gcip job (see: https://github.com/NOAA-EMC/global-workflow/issues/399#issuecomment-923103405). Will need this if supporting WAFS off WCOSS2.

The `util_shared` module is needed on WCOSS2 for the wave bulletins job. See this comment: https://github.com/NOAA-EMC/global-workflow/issues/399#issuecomment-923103405. Will need this installed on Hera/Orion too if we plan to support the wave bulletins job off WCOSS2.